internal/device/darwin.go

```diff
@@ -44,3 +65,31 @@ func Notify() {
 		log.Fatal(err)
 	}
 }
+
+// utils
+
+func isCharged(status string) bool {
+	return strings.EqualFold(status, "charged")
+}
+
+func is100Percent(percent string) bool {
+	return strings.EqualFold(percent, "100%")
+}
+
+func isFinishingCharge(status string) bool {
+	return strings.EqualFold(status, "finishing charge")
+}
+
+func isCharging(status string) bool {
+	return strings.EqualFold(status, "charging")
+}
+
+func isGte95(percent string) bool {
+	pct, err := strconv.Atoi(strings.Split(percent, "%")[0])
+
+	if err == nil {
```

Comment:
We're so used to seeing `if err != nil` that it would be very easy to misread this. So how about:

```suggestion
	if err != nil {
		return false
	}
	return pct >= 95
```

Reply:
Agreed.  Resolved in f72e0d0ca713cd763c8bb6a4447a856edc65e35b